cmake/public/LoadHIP.cmake

```diff
@@ -156,15 +156,35 @@ find_package_and_print_version(HIP 1.0)
 
 if(HIP_FOUND)
   set(PYTORCH_FOUND_HIP TRUE)
+  set(FOUND_ROCM_VERSION_H FALSE)
+
+  if(EXISTS ${ROCM_PATH}/.info/version-dev)
+    # ROCM < 4.5, we don't have the header api file, use flat file
+    file(READ "${ROCM_PATH}/.info/version-dev" ROCM_VERSION_DEV_RAW)
+    message("\n***** ROCm version from ${ROCM_PATH}/.info/version-dev ****\n")
+  endif()
```

Comment:
We should be able to get rid of this section as well, since this file is not relevant for newer ROCm versions.